libpod/oci_conmon_linux.go

```diff
@@ -162,10 +162,7 @@ func (r *ConmonOCIRuntime) moveConmonToCgroupAndSignal(ctr *Container, cmd *exec
 	}
 
 	/* We set the cgroup, now the child can start creating children */
-	if err := writeConmonPipeData(startFd); err != nil {
```

Comment:
I think there's a linter to catch such cases.  Not sure whether it's enabled though.

Reply:
clearly not otherwise it should have flagged this one